src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -60,24 +60,24 @@
 ;;; keep these overrides.
 (def tls-1-dot-3-ciphers
   ["TLS_AES_128_GCM_SHA256"
-   "TLS_AES_256_GCM_SHA384"
-   "TLS_CHACHA20_POLY1305_SHA256"])
+   "TLS_AES_256_GCM_SHA384"])
 
 (def tls-1-dot-2-ciphers
-  ["TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256"
-   "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256"
+  ["TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384"
    "TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384"
-   "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384"
-   "TLS_DHE_RSA_WITH_AES_128_GCM_SHA256"
-   "TLS_DHE_RSA_WITH_AES_256_GCM_SHA384"])
+   "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256"
+   "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256"
+   "TLS_DHE_RSA_WITH_AES_256_GCM_SHA384"
+   "TLS_DHE_RSA_WITH_AES_128_GCM_SHA256"])
 
 (def acceptable-ciphers
   (into [] (concat tls-1-dot-3-ciphers tls-1-dot-2-ciphers)))
 
 (def fips-tls-1-dot-2-ciphers
-  ["TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256"
```

Comment:
I thought this work was to remove ChaCha20, what's up with the rest of the changes? (should also be in the commit message)

Reply:
Sorry, you're right, I rushed through this one.  Let me add some notes.